extensions/lorem-ipsum/src/utils.tsx

```diff
@@ -1,6 +1,9 @@
-import { closeMainWindow, copyTextToClipboard, pasteText, showHUD } from "@raycast/api";
+import { closeMainWindow, Clipboard, showHUD } from "@raycast/api";
 import { LoremIpsum } from "lorem-ipsum";
 
+// don't want to cause a heap error, so cap it 😱
+const LOREM_IPSUM_MAX_NUMBER = 10000;
```

Comment:
If this is a risk, perhaps 10k is still a bit high?

Reply:
Simple enough to reduce - whatever you feel approproate/realistic without being restrictive.

Reply:
Might also depend on the command, what about 2k then? Most users will do 1 anyways 😊